pipeline-modules/infra-service/azure/terraform.yaml

```diff
@@ -135,16 +135,18 @@ template: |
   {%- endmacro %}
 
   steps:
+  - task: ms-devlabs.custom-terraform-tasks.custom-terraform-installer-task.TerraformInstaller@0
```

Comment:
previously i faced the issue with azure saying `-script` should be the first tag. please double check on that.

Reply:
I ran into many issues with this which are VAN-2346 - I tested this PR with my changed on VAN-2346 and it works.